Rocket.Chat/Views/Chat/New Chat/ChatItems/BaseMessageChatItem.swift

```diff
@@ -22,10 +23,11 @@ class BaseMessageChatItem {
         return RCDateFormatter.time(date)
     }
 
-    init(user: UnmanagedUser?, avatar: String?, emoji: String?, date: Date?, isUnread: Bool = false) {
+    init(user: UnmanagedUser?, avatar: String?, emoji: String?, alias: String?, date: Date?, isUnread: Bool = false) {
```

Comment:
What about doing the same thing here as you did in the other places?

Reply:
Yes! :-)